qemu/tests/block_copy.py

```diff
@@ -428,3 +428,44 @@ def verify_md5(self, file_name):
         if status != 0:
             raise error.TestFail("File %s changed, md5sum check output: %s" %
                                  (file_name, output))
+
+    def mount_disks(self):
+        """
+        Mount added disks inside guest.
+        """
+        os_type = self.params["os_type"]
+        if os_type == "linux":
```

Comment:
why not judge the os_type when call mount_disks()?

Reply:
1. Based on you current implantation, the function should named `mount_disks`, it might be good to print a log when `os_type = windows"`, so everything is presented explicitly.

2. if you decide to name it  `mount_disks_linux`(which I think should be `mount_linux_disks`, to be consistent with `format_linux_disk`)， I don't think you
need the if clause inside the method.